extensions/interactions/base_test.py

```diff
@@ -295,11 +295,11 @@ def test_default_interactions_are_valid(self):
             # inline ones do not.
             if interaction.display_mode == base.DISPLAY_MODE_INLINE:
                 self.assertIsNone(interaction.instructions)
+                self.assertIsNone(interaction.narrow_instructions)
             else:
                 self.assertTrue(
                     isinstance(interaction.instructions, basestring))
                 self.assertIsNotNone(interaction.instructions)
```

Comment:
You'd need to add a self.assertIsNotNone(interaction.narrow_instructions) here, right?